src/MongoDB.Driver/Encryption/MongocryptdFactory.cs

```diff
@@ -165,6 +165,33 @@ private bool ShouldMongocryptdBeSpawned(out string path, out string args)
             }
 
             return false;
+
+            string GetMongocryptdExtension()
+            {
+                var currentOperatingSystem = OperatingSystemHelper.CurrentOperatingSystem;
+                switch (currentOperatingSystem)
+                {
+                    case OperatingSystemPlatform.Windows:
+                        // "nul" is the windows specific value.
+                        return ".exe";
+                    case OperatingSystemPlatform.Linux:
+                    case OperatingSystemPlatform.MacOS:
+                    default: return string.Empty;
+                }
+            }
+
+            string GetLogPath()
+            {
+                var currentOperatingSystem = OperatingSystemHelper.CurrentOperatingSystem;
+                switch (currentOperatingSystem)
+                {
+                    case OperatingSystemPlatform.Windows: return "nul";
+                    // Unix - based platforms should use "/dev/null"
+                    case OperatingSystemPlatform.Linux:
+                    case OperatingSystemPlatform.MacOS:
+                    default: return "/dev/null";
```

Comment:
`return` statements should be on their own lines.

Reply:
done